src/icons/package.json

```diff
@@ -8,7 +8,6 @@
     "url": "https://github.com/NYPL/nypl-design-system/tree/development/src/icons"
   },
   "scripts": {
-    "prepare": "npm run build",
```

Comment:
this still needs to be removed?

Reply:
Yeah. I can try to see what Travis thinks and then revert if it fails.

Reply:
🎉